packages/runtime/src/core.ts

```diff
@@ -482,6 +482,20 @@ export class FederationHost {
     );
   }
 
+  initRawContainer(
+    name: string,
+    url: string,
+    container: RemoteEntryExports,
+  ): Module {
+    const remoteInfo = getRemoteInfo({ name, entry: url });
+    const module = new Module({ host: this, remoteInfo });
```

Comment:
@2heal1, could you give me your input on this solution? I believe there are relevant use cases for when a user may sideload a container, and we need a way to convert it to a module instance.

Reply:
Not encounter the case , but I think this solution is okay :D